TarponSimulator2017/Core/FishingRod.cs

```diff
@@ -14,17 +16,29 @@ public class FishingRod : GameObject, IUpdatable
 		/// <value>The fishing float.</value>
 		public FishingFloat FishingFloat { get; private set; }
 
+		/// <summary>
+		/// Gets the list of fishes.
+		/// </summary>
+		/// <value>The list of fishes.</value>
+		public IList<Fish> ListOfFishes { get; private set; }
+
+		/// <summary>
+		/// Gets or sets the caught fish.
+		/// </summary>
+		/// <value>The caught fish.</value>
+		public Fish CaughtFish { get; set; }
+
 		/// <summary>
 		/// Initializes a new instance of the <see cref="Tarpon.Core.FishingRod"/> class.
 		/// </summary>
 		/// <param name="Position">Position. !! Must be given in parent's frame of reference. !!</param>
```

Comment:
If some times we need to give an absolute position, and sometimes a relative position, and there are some chance of being wrong, we may be able to solve this via types and objects, either by:
 * Encapsulating Vector2 in an object AbsolutePosition or RelativePosition and using this types. Functions to get positions will also return these objects, so no possible errors, and it had more "sense" to our program.
 * Or always return an object Position, which contains the object, and which expose two functions: `RelativePosition` and `AbsolutePosition` - so the function can choose and there is only one object position to manipulate.